enginetest/queries/queries.go

```diff
@@ -4612,6 +4612,14 @@ Select * from (
 			{uint64(18446744073709551613)},
 		},
 	},
+	{
+		Query:    "select 0 in (1 / 1000), 0 in (1 / 1000, 0), 0.001 in (1 / 1000, 0), 0.0001 in (1 / 1000, 0);",
```

Comment:
This stops working when the scale is greater than 4
Dolt:
```sql
tmp> select 0 in (1/100000);
+-----------------+
| 0 in (1/100000) |
+-----------------+
| true            |
+-----------------+
1 row in set (0.00 sec)
```

MySQL
```sql
mysql> select 0 in (1/100000);
+-----------------+
| 0 in (1/100000) |
+-----------------+
|               0 |
+-----------------+
1 row in set (0.0004 sec)
```

However, MySQL does start to break around `100 000 000 000` (scale 11)
```
mysql> select 0 in (1/100000000000);
+-----------------------+
| 0 in (1/100000000000) |
+-----------------------+
|                     1 |
+-----------------------+
1 row in set (0.0004 sec)

mysql> select 0 = (1/100000000000);
+----------------------+
| 0 = (1/100000000000) |
+----------------------+
|                    1 |
+----------------------+
1 row in set (0.0004 sec)
```

It looks like MySQL has some other internal logic dealing with decimal precision.
I think this should be dealt with in another PR; a skipped test or issue is good enough for now.

Reply:
tracking issue: https://github.com/dolthub/dolt/issues/7207